src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```

Comment:
can we deal with it and remove this? I don't understand what it wants to say :)

Reply:
This files and the one below were actually examples from Storybook. I forgot to remove them. Sorry about that :confused: